examples/convolution/convolution.cc

```diff
@@ -65,7 +65,7 @@ int main(int argc, char* argv[]) {
 		celerity::accessor gauss{gaussian_mat_buf, cgh, celerity::access::all{}, celerity::read_only};
 		celerity::accessor out{image_tmp_buf, cgh, celerity::access::one_to_one{}, celerity::write_only, celerity::no_init};
 
-		cgh.parallel_for<class gaussian_blur>(celerity::range<2>(image_height, image_width), [=, fs = filter_size](celerity::item<2> item) {
```

Comment:
Should we remove *all* kernel names in the examples? They are optional now, but they do serve some useful documentation purpose in some cases IMHO (i.e. if a kernel is called `gaussian_blur` or `sharpen` you can more quickly recognize what it's supposed to do).

Reply:
Good point, also useful in graph printing.

Reply:
Many of these example kernels are the only `submits` from a function that already aptly names them, but I agree it makes graph printing less descriptive. I dropped this change until we either come up with an infrastructure that does not rely on types for kernel names (think `celerity::debug::set_cgf_name(cgh)`) or we use C++20 `std::source_location` to name kernels based on their surrounding function.